src/dcqc/parsers.py

```diff
@@ -0,0 +1,50 @@
+import csv
+from collections.abc import Collection, Iterator
+from pathlib import Path
+from typing import Optional
+
+from dcqc.file import File
+from dcqc.suites.suite_abc import SuiteABC
+from dcqc.target import Target
+
+
+class CsvParser:
```

Comment:
I like this a lot.  In general, I think that most of the modules you have contributed in this suite of PR's could have docstrings so that people can go to the module and have an understanding on a high level of the classes and functions the module contains.

That said, you have done a wonderful job of naming each of your modules.

Reply:
It depends on need and alternative implementations. For example, instead of implementing a `FileViewParser`, we would add support for file views to SynapseFS, which can return them as CSV files. In this case, we could just re-use CsvParser.